### PR TITLE
Fix for HTML Pipeline 2.2.0 and 2.2.1

### PR DESCRIPTION
**Pull request by benbalter**
Which fixes https://github.com/jch/html-pipeline/pull/230.

I believe it technically works in < 2.0 and >= 2.2.2.

### CODE REVIEW

File: jemoji.gemspec [hunk 1]
```diff
@@ -10,7 +10,7 @@ Gem::Specification.new do |s|
   s.files    = [ 'lib/jemoji.rb' ]
 
   s.add_dependency 'jekyll', '>= 2.0'
-  s.add_dependency 'html-pipeline', '~> 1.9'
+  s.add_dependency 'html-pipeline', '~> 2.2.2'
```
Comment (parkr) [inline jemoji.gemspec]: This won't allow us to use 2.3 or higher. Is that intended? This says `>= 2.2.2, < 2.3.0`

**Comment by parkr** [comment]
@benbalter Conflicts here with #26. Please see my comment above, too. :smile:

File: jemoji.gemspec (hunk 1 above)
Comment (benbalter) [inline jemoji.gemspec]: I guess `>= 2.2.2` is what we want?

**Comment by benbalter** [comment]
Resolved the merge conflict, but not sure the best way to handle the dependency, halp?

**Comment by parkr** [comment]
> not sure the best way to handle the dependency, halp?

What are you trying to do? Say `2.x as long as it's after 2.2.2`?

**Comment by benbalter** [comment]
> What are you trying to do? Say 2.x as long as it's after 2.2.2?

Yes, or technically `< 2.0`, but that felt too complicated.

**Comment by parkr** [comment]
Well `~> 2.2` will allow 2.2.0 or higher until 3.0. Would that work?

**Comment by benbalter** [comment]
> Well ~> 2.2 will allow 2.2.0 or higher until 3.0. Would that work?

No, the issue is 2.2.0 and 2.2.1 have [a regression that breaks the build](https://github.com/jch/html-pipeline/pull/230).

**Comment by benbalter** [comment]
One option would be to pass four `nil`s to `mention_link_filter` which should also fix it for 2.2.0 and 2.2.1.. probably easier on the user if we do it that way.

**Comment by benbalter** [comment]
JK again. I thought this was `jekyll-mentions` I think this should be fine, will test, and reopen if not the case (but the same discussion will apply to `jekyll-mentions`... going to fix there).

**Pull request closed without merging**